### DISCUSSION TITLE
Variable expansion in computations

### DISCUSSION THREAD

**Discussion by mavam** [quarto-dev/quarto-cli#4410] [Q&A] [answered]
Is it possible to expand a variable from `_variables.yml` inside a computation?

I have something like this:

    ```{bash}
    cmd --param={{<var magic>}}
    ```
The expansion only works outside the cell. Is there away to pass the parameter inside, e.g., as argument to `{bash}`?

**Comment by mcanouil** [comment, marked as answer]
For clarity, it's a shortcode and proper syntax is `{{< KEY ARG >}}`.

This being said, the only variables you can use are the one under the `params` YAML key, see <https://quarto.org/docs/computations/parameters.html>.

There are several GitHub Discussions about this if you want to know more about it and workarounds.
Quarto team has plan to allow to use variables and/or environment variables in code cells, but this is something for the future, so you'll have to be patient.

**Reply by mavam** [reply]
Thanks, I guess I'll resort to `params`.
